schema-loader/src/main/java/com/scalar/db/schemaloader/command/CassandraCommand.java

```diff
@@ -99,13 +96,12 @@ public Integer call() throws Exception {
       metaOptions.put(CassandraAdmin.REPLICATION_FACTOR, replicationFactor);
     }
 
-    SchemaOperator operator = SchemaOperatorFactory.getSchemaOperator(props);
-    List<Table> tableList = SchemaParser.parse(schemaFile.toString(), metaOptions);
+    SchemaOperator operator = SchemaOperatorFactory.getSchemaOperator(props, false);
 
     if (deleteTables) {
-      operator.deleteTables(tableList);
+      operator.deleteTables(schemaFile, metaOptions);
     } else {
-      operator.createTables(tableList, metaOptions);
+      operator.createTables(schemaFile, metaOptions);
     }
 
     operator.close();
```

Comment:
It's not directly related to this PR, but can you please also do this close in a `finally` block? Also please do the same for the other storage specific commands?

Reply:
Added it in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/0c8f3fe47b504108058a0f419a42bb8a6808f80d)